test/controllers/RecordsControllerSpec.scala

```diff
@@ -2,36 +2,80 @@ package controllers
-        new GraphQLConfiguration(app.configuration), getValidKeycloakConfiguration)
-      val recordsPage = controller.recordsPage(UUID.randomUUID()).apply(FakeRequest(GET, "/consignment/123/records"))
+    "render the records page with progress bar after initial upload is complete and user clicks continue" in {
+      val graphQLConfiguration = new GraphQLConfiguration(app.configuration)
+      val consignmentService = new ConsignmentService(graphQLConfiguration)
+      val client = new GraphQLConfiguration(app.configuration).getClient[fileCheck.Data, fileCheck.Variables]()
+
+      val filesProcessed = 1
+      val antivirusProgress = fileCheck.GetConsignment.FileChecks.AntivirusProgress(filesProcessed);
+      val fileChecks = fileCheck.GetConsignment.FileChecks(antivirusProgress)
+      val data: client.GraphqlData = client.GraphqlData(Some(fileCheck.Data(Some(fileCheck.GetConsignment(totalFiles, fileChecks)))))
+      val dataString: String = data.asJson.printWith(Printer(dropNullValues = false, ""))
+
+      wiremockServer.stubFor(post(urlEqualTo("/graphql"))
+        .willReturn(okJson(dataString)))
+
+      val recordsController = new RecordsController(
+        getAuthorisedSecurityComponents,
+        new GraphQLConfiguration(app.configuration),
+        getValidKeycloakConfiguration,
+        consignmentService
+      )
+
+      val recordsPage = recordsController.recordsPage(consignmentId).apply(FakeRequest(GET, s"consignment/${consignmentId}/records"))
+
       playStatus(recordsPage) mustBe OK
       contentType(recordsPage) mustBe Some("text/html")
-      contentAsString(recordsPage) must include ("records.header")
-      contentAsString(recordsPage) must include ("records.title")
+      contentAsString(recordsPage) must include("checkingRecords.header")
+      contentAsString(recordsPage) must include("checkingRecords.title")
+      contentAsString(recordsPage) must include("progress")
     }
 
+
     "return a redirect to the auth server with an unauthenticated user" in {
+      val graphQLConfiguration = new GraphQLConfiguration(app.configuration)
+      val consignmentService = new ConsignmentService(graphQLConfiguration)
       val controller = new RecordsController(getUnauthorisedSecurityComponents,
-        new GraphQLConfiguration(app.configuration), getValidKeycloakConfiguration)
-      val recordsPage = controller.recordsPage(UUID.randomUUID()).apply(FakeRequest(GET, "/consignment/123/records"))
-      redirectLocation(recordsPage).get must startWith ("/auth/realms/tdr/protocol/openid-connect/auth")
+        new GraphQLConfiguration(app.configuration), getValidKeycloakConfiguration, consignmentService)
+      val recordsPage = controller.recordsPage(UUID.randomUUID()).apply(FakeRequest(GET, s"/consignment/${consignmentId}/records"))
+      redirectLocation(recordsPage).get must startWith("/auth/realms/tdr/protocol/openid-connect/auth")
       playStatus(recordsPage) mustBe FOUND
```

Comment:
Really minor thing, but I think it's worth swapping the order of the redirect location and HTTP status assertions.

If the test fails because the method returns a different status, it will be easier to debug if the status check is run first (because the message will be something like "expected FOUND but got UNAUTHORIZED" rather than the `NoSuchElementException` that you'll get from the redirect location assertion.